src/App.tsx

```diff
@@ -183,9 +190,9 @@ function getMessage(logging: JSON): undefined | getMessageReturn {
             app_name.push(logs[msg]["message"]["application_name"]); 
             const level = logs[msg]["message"]["level"];
             const log_message = logs[msg]["message"]["full_message"];
-            const log_level_str = getLogLevel(level);
+            const log_level_str = log_levels[level] || "UNKNOWN";
             debug.push(log_level_str);
-            message.push(log_message);
+            message.push(log_message)
```

Comment:
Probably best to leave it on for consistency with the rest of your code

```suggestion
            message.push(log_message);
```